yt_dlp/options.py

```diff
@@ -440,8 +441,24 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help='Do not mark videos watched (default)')
     general.add_option(
         '--no-colors', '--no-colours',
-        action='store_true', dest='no_color', default=False,
-        help='Do not emit color codes in output (Alias: --no-colours)')
+        action='store_const', dest='color', const={
+            'stdout': 'no_color',
+            'stderr': 'no_color',
+        },
+        help=optparse.SUPPRESS_HELP)
+    general.add_option(
+        '--color',
+        dest='color', metavar='[STREAM:]POLICY', default={}, type='str',
+        action='callback', callback=_dict_from_options_callback,
+        callback_kwargs={
+            'allowed_keys': 'stdout|stderr',
+            'default_key': ['stdout', 'stderr'],
+            'process': str.strip,
+        }, help=(
+            'A mapping with output stream names as keys and their respective color policy as values. '
+            'Can also just be a single color policy, in which case it applies to all outputs. '
+            'STREAM is one of "stdout" and "stderr". '
+            'POLICY is one of "always", "auto" (default), "no_color" and "never"'))
```

Comment:
```suggestion
            'Whether to emit color codes in output, optionally prefixed by '
            'the STREAM (stdout or stderr) to apply the setting to. '
            'Can be one of "always", "auto" (default), "never", or '
            '"no_color" (support terminal sequences, but not color)'))
```

Reply:
I don't particularly like "(support terminal sequences, but not color)", but idk how else to describe it.

Reply:
Something like "Strip only color sequences", "Avoid only color sequences"

Reply:
Since the option name is `--color`, "avoid color sequences" won't be clear on what is is allowing.

Reply:
How about this version